services/core-api/app/api/mines/permits/permit_extraction/create_permit_condition_report_requirement.py

```diff
@@ -71,13 +74,17 @@ def create_permit_condition_report_requirement(
     # Create the MineReportPermitRequirement
     mine_report_permit_requirement = MineReportPermitRequirement(
         report_name=report_name,
-        permit_condition_id=condition.permit_condition_id,
         permit_amendment_id=task.permit_amendment.permit_amendment_id,
         cim_or_cpo=cim_or_cpo,
         due_date_period_months=due_date_period_months or 0,
         initial_due_date=initial_due_date,
         ministry_recipient=None,  # Not specified in permits themselves.
     )
+    xref = MineReportReqPermitConditionXref(
+        mine_report_permit_requirement=mine_report_permit_requirement,
+        permit_condition_id=condition.permit_condition_id
+    )
+    mine_report_permit_requirement.mine_report_req_permit_conditions.append(xref)
```

Comment:
haven't really tested this out (beyond running existing tests), but it is quite possible that the extraction part of it is done. I just wanted the tests to pass...